client/components/character/addEditModal.vue

```diff
@@ -200,7 +200,18 @@
                 get() { return !!this.value; },
                 set() { /* We ignore setting */ }
             },
-            systems() { return this.allSystems.filter((sys) => sys.disabled !== true); },
+            systems()
+            {
+                return this.allSystems
+                    .filter((sys) => sys.disabled !== true)
```

Comment:
Is this line actually doing anything? Seems all other references to disabled are changed to use status.

Reply:
Nope, you're right! This should drop out.